entrypoint.sh

```diff
@@ -14,6 +14,10 @@ auth_type=$CONNECT
 managed_variables_var="OP_MANAGED_VARIABLES"
 IFS=','
 
+if [[ "$OP_CONNECT_HOST" != "http://"* ]] && [[ "$OP_CONNECT_HOST" != "https://"* ]]; then
+  export OP_CONNECT_HOST="http://"$OP_CONNECT_HOST
+fi
```

Comment:
One of the main reasons to use the CLI in the action is to get local vs. CI parity. Adding custom environment variable logic in the action alone would be a step backwards.

Also raises the question, why `http` and not `https`? The answer is probably: we should let the user decide that and not make any assumptions here.

If we do really want to add this, we should probably do that on the CLI side and bump the CLI version in the action.